bitcoind/config.yml

```diff
@@ -1,8 +1,10 @@
 versions:
-  "0.21.1":
+  "23.0":
     build_args:
-      BITCOIN_VERSION: 0.21.1
-      BITCOIN_SHA256: 366eb44a7a0aa5bd342deea215ec19a184a11f2ca22220304ebb20b9c8917e2b
+      BITCOIN_VERSION: 23.0
+      SHA256_ARM64: 06f4c78271a77752ba5990d60d81b1751507f77efda1e5981b4e92fd4d9969fb
+      SHA256_AMD64: 2cca490c1f2842884a3c5b0606f179f9f937177da4eadd628e3f7fd7e25d26d0
     test_config:
       cmd:
         - bitcoind --version
+    platforms: *base_platforms
```

Comment:
Here you can have a 23.0_amd and a 23.0_arm config with different SHAs

Reply:
Not need, it is when you use this image in docker run or docker service , Docker picks the correct image based on the node's platform by setting the --platform flag to specify the target platform for the build output, (for example, linux/amd64, linux/arm64, or darwin/amd64). For more information you can see this link https://docs.docker.com/buildx/working-with-buildx/

Reply:
This was in reference of this comment: https://github.com/ekino/docker-buildbox/pull/764/files#r867968980